tool/ci.sh

```diff
@@ -47,11 +47,11 @@ for PKG in ${PKGS}; do
     exit 64
   fi
 
-  pub get --no-precompile || EXIT_CODE=$?
+  dart pub get --no-precompile || EXIT_CODE=$?
```

Comment:
Top of this file says: `# Created with package:mono_repo v3.4.2`

I'm guessing there is little reason to modify it, maybe we should patch `mono_repo` to write more clearly not to modify such files, I assume `mono_repo` will overwrite it.. If not then changes in this file might make sense...

Reply:
Should I discard this changes? @jonasfj

Reply:
yeah, let's not modify generated files.

Reply:
ok sure Thanks.

Reply:
Done @jonasfj